infrastructure/output.tf

```diff
@@ -9,3 +9,15 @@ output "vaultName" {
 output "microserviceName" {
   value = "${var.component}"
 }
+
+output "TEST_SCAN_DELAY" {
+  value = "${var.scan_delay}"
+}
+
+output "TEST_STORAGE_ACCOUNT_NAME" {
+  value = "${azurerm_storage_account.provider.name}"
+}
+
+output "TEST_STORAGE_ACCOUNT_KEY" {
+  value = "${azurerm_storage_account.provider.primary_access_key}"
+}
```

Comment:
I am not sure I understand the question. Keep in mind  that this part is not my code. I reintroduced it after Leszek reverted a previous merge that was causing issues. Can you please elaborate?

Reply:
I am just guessing. at least storage account name should work. Since it is only related for test container, is prefixed with `TEST_` too, figured it would be good to use `test` container's module. hence suggestion `"${azurerm_storage_container.test.storage_account_name}"`. feel free to ignore because currently it works so why break the build. it would still need same values so not much of a point in change